internal/data/payments.go

```diff
@@ -491,7 +491,7 @@ func (p *PaymentModel) GetReadyByDisbursementID(ctx context.Context, sqlExec db.
 }
 
 func (p *PaymentModel) GetReadyByID(ctx context.Context, sqlExec db.SQLExecuter, paymentIDs ...string) ([]*Payment, error) {
-	query := fmt.Sprintf(getReadyPaymentsBaseQuery, "AND p.id = ANY($4)")
+	query := fmt.Sprintf(getReadyPaymentsBaseQuery, "AND p.id = ANY($4) ORDER BY p.updated_at ASC")
```

Comment:
❓ Given that this does a `GetByID` (returns 0 or 1 payment), do we need the "Order By" ?

Reply:
The method actually allows for fetching multiple payments in ready state, given a list of IDs. One of the tests failed a couple of times because the returned payments were not always in the expected order:

https://github.com/stellar/stellar-disbursement-platform-backend/blob/9a449a74d04c89f9f811c82acfef381e6ff8f3fa/internal/data/payments_test.go#L1353